tests/cupyx_tests/scipy_tests/fftpack_tests/test_fftpack.py

```diff
@@ -578,3 +579,30 @@ def test_irfft_overwrite(self, xp, scp, dtype):
         x = testing.shaped_random(self.shape, xp, dtype)
         return scp.fftpack.irfft(x, n=self.n, axis=self.axis,
                                  overwrite_x=True)
+
+
+@testing.parameterize(
+    {'shape': (32, 16, 4), 'data_order': 'F'},
+    {'shape': (4, 32, 16), 'data_order': 'C'},
+)
+class TestFftnView(unittest.TestCase):
```

Comment:
I suppose the test is added in this file because it's for where `get_fft_plan()` resides?

Reply:
yes. I started out with it in the core cupy fft tests, but moved it here since fftpack contained the planning functionality.